lib/vdc/DCCF.cpp

```diff
@@ -418,6 +418,40 @@ int DCCF:: _get_latlon_coordvars(
 	return(0);
 }
 
+bool DCCF::_isUniform(NetCDFCFCollection *ncdfc, string varname) {
+	vector <size_t> dims = ncdfc->GetDims(varname);
+
+	if (dims.size() != 1) return(false);
+
+	if (dims[0] == 2) return(true);
+
+	vector <float> buf(dims[0]);
+
+	bool enabled = EnableErrMsg(false);
+	int fd = ncdfc->OpenRead(0, varname);
+	if (fd < 0) {
+		EnableErrMsg(enabled);
+		return(false);
+	}
+	int rc = ncdfc->Read(buf.data(), fd);
+	if (rc < 0) {
+		EnableErrMsg(enabled);
+		return(false);
+	}
+	ncdfc->Close(fd);
+	EnableErrMsg(enabled);
+
+	float epsilon = 0.0001;
+	for (int i = 0; i<buf.size()-2; i++) {
+		if (! Wasp::NearlyEqual((buf[i+1] - buf[i]), (buf[i+2] - buf[i+1]), epsilon)) {       
+			return(false);
+		}   
```

Comment:
This can be simplified and made more accurate by comparing the first delta against all others rather than all neighboring deltas.

Reply:
True enough. Done.